tests/test_link_io_structure.py

```diff
@@ -0,0 +1,121 @@
+from symlink_komodo.sanity_check import read_link_structure, compare_links, check_root_nodes, verify_integrity
+from symlink_komodo.create_links import create_symlinks
+
+import os
+from shutil import copy
```

Comment:
No reason to drop the entire namespace here. I think `import shutil` followed by calling `shutil.copy` is a better solution...

Reply:
:+1: